storybook/storybook/src/packages/Form/Input/checkbox/src/CheckboxInput.stories.js

```diff
@@ -136,7 +136,8 @@ stories.push({
 
 const storyData = {
   name: 'Form.Input.Checkbox',
-  docs:readme,
+  docs: readme,
```

Comment:
Why insert readme inside "readme" and "docs"?

Reply:
In many components, the readme was included in the docs part.
I did not know if I had to keep together.
So, I insert the readme in both.

Reply:
Do you want to remove all readme in docs ?
Or do you want to remove the docs completly ?

Reply:
ok i delete readme tab, just keep the docs below the demo.